src/cacheAdapterEnhancer.ts

```diff
@@ -78,3 +78,9 @@ export default function cacheAdapterEnhancer(adapter: AxiosAdapter, options: Opt
 		return adapter(config);
 	};
 }
+
+declare module 'axios' {
```

Comment:
very useful improvement! could u pls move this declaration into src/typings.d.ts

Reply:
Hi, moving it to typings.d.ts, breaks declaration of buildUrl augmentations: 
`src/typings.d.ts(15,16): error TS2665: Invalid module name in augmentation. Module 'axios/lib/helpers/buildURL' resolves to an untyped module at 'E:/personal/hub/axios-extensions/node_modules/axios/lib/helpers/buildURL.js', which cannot be augmented.
src/utils/buildSortedURL.ts(7,22): error TS7016: Could not find a declaration file for module 'axios/lib/helpers/buildURL'. 'E:/personal/hub/axios-extensions/node_modules/axios/lib/helpers/buildURL.js' implicitly has an 'any' type.
  Try `npm install @types/axios` if it exists or add a new declaration (.d.ts) file containing `declare module 'axios';`
Didn't find a way yet to fix it.

Reply:
Found this comment about typings module declaration: https://stackoverflow.com/a/42522751/410893